exercises/finaleworkflow/README.md

```diff
@@ -0,0 +1,49 @@
+# Exercise 4: Finale Workflow
+
+One of the benefits of Temporal is that it provides SDKs for several
+languages and you can use multiple languages in the context of a single
+Workflow. For example, you might write your Workflow in TypeScript but use
+Java or Go for an Activity in that workflow.
+
+The last exercise in this workshop does exactly that. The Workflow
+itself is written in TypeScript, but the Activity that is executed as part
+of this Workflow is written in Java, as is the Worker that runs it.
+Since the Activity is written in Java, it's able to use a Java graphics
+library that would otherwise be would be incompatible with a typical
+TypeScript program.
+
+# Start the Worker (Java)
+
+In one terminal, run the following command from the `finaleworkflow` directory:
+
+```
+java -classpath \
+java-activity-and-worker.jar \
+io.temporal.training.PdfCertWorker
+```
+
+# Edit the Workflow (dotnet)
+
+Open the [Program.cs](./src/Client/Program.cs) file, and change the argument passed to the Workflow from 'Maxim Fateev' to your name.
+
+# Start the Workflow (dotnet)
+
+Run the command below from the src directory in another terminal to start the Workflow:
+
+```sh
+dotnet run --project Client/Client.csproj
+```
+
+- Once the Workflow is complete, use the explorer
```

Comment:
This did not result in a PDF (nothing visible happened). 

If your goal is to run a C# Workflow that calls an Activity written in Java, you'd need to start a Worker in C# (in addition to the Java Worker started in the first step).

Reply:
Update: Tom sees his PDF :)